libnmstate/ifaces/ifaces.py

```diff
@@ -198,6 +193,28 @@ def _apply_copy_mac_from(self):
             else:
                 iface.apply_copy_mac_from(port_iface.mac)
 
+    def _validate_copy_mac_from_iface_is_in_port(self, iface):
+        iface_port = iface.port
```

Comment:
Would you create a helper function for the code below that finds the iface_port? It would make the function more readable and you would not need to pre-initialize this variable.

Reply:
Done